src/scf/self_consistent_field.jl

```diff
@@ -96,6 +98,7 @@ function self_consistent_field(ham::Hamiltonian, n_bands;
         Psi, orben, occupation, εF, ρout = next_density(ham, n_bands; Psi=Psi,
                                                         eigensolver=eigensolver, tol=diagtol)
         energies = update_energies(ham, Psi, occupation, ρout)
+        energies[:Entropy] = -model.temperature * compute_entropy(basis, orben, εF=εF)
```

Comment:
For consistency, compute_entropy should match Entropy. So either we call Entropy -TS, or we put the -T into compute_entropy. I'm in favor of option 2 (this will become a Term anyway)

Reply:
I see your point, but I get slight stomach ache from calling "-TS" the entropy ... I'll call it `compute_entropy_term` instead.

Reply:
If this becomes a term, how would you handle the dependency on `ε_F`?

Reply:
Then put the - T in the compute function. Good point on epsf, I'll just pass it as a kwarg

Reply:
Sounds reasonable ...